packages/release-manifests/src/manifest.ts

```diff
@@ -57,6 +58,27 @@ export async function getManifestByVersion(
   return await response.json();
 }
 
+/**
+ * Options for {@link getManifestByFile}.
+ * @public
+ */
+export type GetManifestByFileOptions = {
+  file: string;
+};
+
+/**
+ * Returns a release manifest based on file.
+ * @public
+ */
+export async function getManifestByFile(
```

Comment:
Let's skip this addition to the `release-manifests` package tbh, it's a simple file read that we can do straight in the CLI instead.